image.yaml

```diff
@@ -32,19 +32,19 @@ modules:
           - name: cct_module
             git:
                   url: https://github.com/jboss-openshift/cct_module.git
-                  ref: master
+                  ref: 0.39.0
           - name: jboss-eap-modules
             git:
                   url: https://github.com/jboss-container-images/jboss-eap-modules.git
-                  ref: master
+                  ref: EAP_730_CR1
```

Comment:
Just a note, some commits have been applied to jboss-eap-modules since this tag. I suppose this will be handled in another commit.

Reply:
This was added for CR2